### ISSUE TITLE
Please add lora support for higher ranks and alpha values

### ISSUE THREAD

**Issue by parikshitsaikia1619** [vllm-project/vllm#2847]
ValueError: LoRA rank 64 is greater than max_lora_rank 16.

**Comment by SuperBruceJia** [comment]
Mark

**Comment by Peter-Devine** [comment]
Bump

**Comment by dspoka** [comment]
It's not super well documented but you need to just pass in "-max-lora-rank 64" or whatever when serving since default is 16. 

python -m vllm.entrypoints.openai.api_server --max-lora-rank 64 \
    --model model_name\
    --enable-lora \
    --lora-modules lora-name=lora_path

**Comment by spreadingmind** [comment]
> It's not super well documented but you need to just pass in "-max-lora-rank 64" or whatever when serving since default is 16.
> 
> python -m vllm.entrypoints.openai.api_server --max-lora-rank 64 --model model_name --enable-lora --lora-modules lora-name=lora_path

Thanks for the answer, it helped me as well. For those who use code, it would be here:
```
          llm = LLM(
            model=args.model, tensor_parallel_size=torch.cuda.device_count(), 
            dtype=args.dtype, trust_remote_code=True, enable_lora=True, max_lora_rank=64
        )
```

**Comment by Napuh** [comment]
Both answers work for me, up to rank 64. Rank > 64 is not supported yet.

See #3934

**Comment by patrickrho** [comment]
Can we get Lora rank > 64 supported and merged?

**Comment by kevinjesse** [comment]
Bump. I need adapters that are much, much larger to be supported. Thanks

**Comment by jiangjin1999** [comment]
Is there something special about lora rank >64. Wonder why only <=64 is supported

**Comment by JohnUiterwyk** [comment]
same here, this is a blocker for me

**Comment by Peter-Devine** [comment]
@JohnUiterwyk , has this not been fixed by the suggestions by @dspoka and @spreadingmind ? Their suggestions worked for me.

**Comment by JohnUiterwyk** [comment]
No, as the maximum max_lora_rank is 64, going higher than that throws an error. I have adapters with rank 128 and 256 for certain uses cases, and can not serve them with vllm as a result of the hardcoded limit for the allowed value passed to max_lora_rank

**Comment by AntreasAntonio** [comment]
Any updates on this? Recent papers showed that a rank=256 seems to be very beneficial for example. I suspect this trend will continue to be the case and increasing in the near future.

**Comment by Jiaxin-Wen** [comment]
> Any updates on this? Recent papers showed that a rank=256 seems to be very beneficial for example. I suspect this trend will continue to be the case and increasing in the near future.

interesting, do you know the paper link @AntreasAntonio

**Comment by Vaibhav-Sahai** [comment]
wanted to update: LORA rank 256 works with vllm 0.8.2
